Xamarin.Forms.Platform.Android/CollectionView/ItemsViewRenderer.cs

```diff
@@ -227,13 +227,19 @@ protected virtual void UpdateItemsSource()
 			UpdateEmptyView();
 		}
 
-		protected virtual void UpdateAdapter()
+		protected virtual ItemsViewAdapter CreateAdapter()
+		{
+			return new ItemsViewAdapter(ItemsView);
+		}
+
+		void UpdateAdapter()
```

Comment:
Changing this one might be breaking for custom renderers. But maybe that’s OK since it’s still preview? @samhouts

Reply:
Yes, since it's in preview, this should be OK from that perspective

Reply:
Yep, we'll just add the `breaking` tag to give users a heads-up in the release notes. Thanks!